uber/site_sections/accounts.py

```diff
@@ -99,6 +99,54 @@ def bulk(self, session, department_id=None, **params):
             'attendees': attendees
         }
 
+    def access_groups(self, session, message='', **params):
+        if 'id' in params and params['id']:
+            access_group = session.access_group(params['id'])
+        else:
+            access_group = AccessGroup()
+
+        if cherrypy.request.method == "POST":
+            access_group.name = params['name']
+
+            for key in params:
+                if key.endswith('_read_only_access'):
+                    col_key = key[:-17]
+                    if params[key] != "0":
+                        access_group.read_only_access[col_key] = params[key]
+                    elif col_key in access_group.read_only_access:
+                        del access_group.read_only_access[col_key]
+                elif key.endswith('_access'):
+                    col_key = key[:-7]
+                    if params[key] != "0":
+                        access_group.access[col_key] = params[key]
+                    elif col_key in access_group.access:
+                        del access_group.access[col_key]
+
+            session.add(access_group)
+            message = check(access_group) or ''
```

Comment:
Oh right, I wanted to add a model check to make sure read only access was never lower than read-write access (which makes no sense). I'll do that in the next commit.